config/Dockerfiles/tests.d/gcloud/03_gcloud_gcdns_zone

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash -xe
 
 # Verify the gcloud gcdns_zone provisioning
-# distros.exclude: centos7 fedora28
-# providers.include: gcloud
+# distros.exclude: centos7 fedora29 fedora30
```

Comment:
Why are we disabling this test?

Reply:
GCDNS zone is already provisioned in elb test that is the reason we are disabling the test